content/hi/docs/tutorials/kubernetes-basics/explore/explore-intro.html

```diff
@@ -0,0 +1,143 @@
+---
+title: पॉड्स और नोड्स देखना
+weight: 10
+---
+
+<!DOCTYPE html>
+
+<html lang="hi">
+
+<body>
+
+<link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+
+
+<div class="layout" id="top">
+
+    <main class="content">
+
+        <div class="row">
+
+     <div class="col-md-8">
+          <h3>उद्देश्य</h3>
+                <ul>
+                    <li>कुबेरनेट्स पॉड्स के बारे में जानें।</li>
+                    <li>कुबेरनेट्स नोड्स के बारे में जानें।</li>
+                    <li>डिप्लॉयड एप्लीकेशन का समस्या निवारण करें।</li>
+                </ul>
+            </div>
+
+            <div class="col-md-8">
+                <h2>कुबेरनेट्स पॉड्स</h2>
+                <p>जब आपने मॉड्यूल <a href="/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/">2</a> में एक एप्लीकेशन बनाया, तो कुबेरनेट्स ने आपके एप्लिकेशन इंस्टेंस को होस्ट करने के लिए एक <b>पॉड</b> बनाया। पॉड एक कुबेरनेट्स एब्स्ट्रैक्शन है जो एक या अधिक एप्लिकेशन कंटेनरों (जैसे डॉकर) के समूह और उन कंटेनरों के लिए कुछ साझा संसाधनों का प्रतिनिधित्व करता है। उन संसाधनों में शामिल हैं:</p>
+                <ul>
+                    <li>साझा स्टोरेज, वॉल्यूम के रूप में</li>
+                    <li>नेटवर्किंग, एक अद्वितीय क्लस्टर IP पते के रूप में</li>
+                    <li>प्रत्येक कंटेनर को चलाने के तरीके के बारे में जानकारी, जैसे कंटेनर इमेज संस्करण या उपयोग करने के लिए विशिष्ट पोर्ट</li>
+                </ul>
+                <p>एक पॉड एक एप्लिकेशन-विशिष्ट "लॉजिकल होस्ट" मॉडल करता है और इसमें विभिन्न एप्लिकेशन कंटेनर हो सकते हैं जो अपेक्षाकृत कसकर युग्मित होते हैं। उदाहरण के लिए, एक पॉड में आपके Node.js ऐप के साथ-साथ एक अलग कंटेनर दोनों शामिल हो सकते हैं जो Node.js वेबसर्वर द्वारा प्रकाशित किए जाने वाले डेटा को फीड करता है। पॉड में कंटेनर एक IP एड्रेस और पोर्ट स्पेस साझा करते हैं जो हमेशा सह-स्थित और सह-अनुसूचित होते हैं, और एक ही नोड पर एक साझा संदर्भ में चलते हैं।</p>
+
+            <p>कुबेरनेट्स प्लेटफॉर्म पर पॉड्स  सबसे छोटे इकाई हैं। जब हम कुबेरनेट्स पर एक डिप्लॉयमेंटडिप्लॉयमेंट बनाते हैं, तो वह डिप्लॉयमेंट अपने अंदर कंटेनरों के साथ पॉड बनाता है (नाकि सीधे कंटेनर). प्रत्येक पॉड उस नोड से जुड़ा होता है जहां यह निर्धारित होता है, और समाप्ति (रीस्टार्ट नीति के अनुसार) या विलोपन तक वहीं रहता है. नोड की विफलता के मामले में, क्लस्टर में अन्य उपलब्ध नोड्स पर समान पॉड्स शेड्यूल किए जाते हैं।</p>
+
+            </div>
+            <div class="col-md-4">
+                <div class="content__box content__box_lined">
+                    <h3>सारांश:</h3>
+                    <ul>
+                        <li>पॉड्स</li>
+                        <li>नोड्स</li>
+                        <li>Kubectl के मुख्य कमांड</li>
```

Comment:
```suggestion
                        <li>kubectl के मुख्य कमांड</li>
```

Reply:
What's wrong with `Kubectl`

Reply:
`kubectl` is a command & its always written that way !! @sftim had also mentioned this in slack

Reply:
I'll leave similar changes for you to have a look into!! So, not marking them in the lines below

Reply:
Kubectl is not used here as a command, we are talking about it as the tool.

Reply:
We should write `kubectl` in lower case whether we mean it as a command you type or as the name of a tool.

Reply:
Oh! Okay